Lesson9/src/TextFormater.java

```diff
@@ -0,0 +1,72 @@
+import java.io.*;
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class TextFormater implements Reader {
+    String str;
+    String[] words;
+    int count1 = 0;
+    int count2 = 0;
+    int count3 = 0;
+
+    @Override
+    public void readFile() {
+        try {
+            String filename;
+            Scanner scanner = new Scanner(System.in);
+            System.out.println("Введите название файла (text.txt):");
+            filename = scanner.nextLine();
+            FileReader fileReader = new FileReader("D://tms2021Android1//Lesson9//" + filename);
+            char[] buf = new char[256];
+            int i;
+            while ((i = fileReader.read(buf)) > 0) {
+                if (i < 256) {
+                    buf = Arrays.copyOf(buf, i);
+                }
+                str = new String(buf);
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    @Override
+    public void splitText() {
+        count1 = 0;
+        count2 = 0;
+        count3 = 0;
+        int sumsentence = 0; // кол-во предложений
+        words = str.split("\\.");
+        for (int i = 0; i < words.length; i++) {
+            sumsentence++;
+        }
+        System.out.println("Кол-во предложений: " + sumsentence);
+        for (String subStr : words) {
+            System.out.println(subStr);
+        }
+    }
+
+    public void chooseFile() throws IOException {
```

Comment:
please separate logic of figuring out if the sentence is valid and logic of writing into files into 2 methods